modules/apps/29-fee/ibc_module.go

```diff
@@ -130,8 +130,13 @@ func (im IBCModule) OnChanCloseInit(
 	portID,
 	channelID string,
 ) error {
-	// TODO: Unescrow all remaining funds for unprocessed packets
+	// delete fee enabled on channel
+	// and refund any remaining fees escrowed on channel
 	im.keeper.DeleteFeeEnabled(ctx, portID, channelID)
+	err := im.keeper.RefundFeesOnChannel(ctx, portID, channelID)
+	if err != nil {
+		panic(err)
```

Comment:
This should only happen if there was a bug in the code and the module account has insufficient funds to refund all escrowed fees. So I panic here, open to a different suggestion on what to do

Reply:
Seems logical to me. Is there any way that the fee's could be refunded just before the channel closes? If so this would panic and we wouldn't call `im.app.OnChanCloseInit`

Reply:
why panic and not just return the error without closing the channel?